install/install.sh

```diff
@@ -98,6 +99,84 @@ install_dashboard(){
     -e SODA_GRAFANA_PORT=3000 \
     -e STORAGE_SERVICE_PLAN_ENABLED=true sodafoundation/dashboard:$DASHBOARD_RELEASE_VERSION  > /dev/null
 }
+
+install_srm_toolchain() {
+    install $@
+    mkdir -p /opt/srm-toolchain
```

Comment:
/opt/srm-toolchain needs to be configurable

Reply:
It is configurable and is picked from the variable